editoast/src/generated_data/error/routes.rs

```diff
@@ -157,7 +157,11 @@ fn check_path(
     // Search for switches out of the path
     let mut res = vec![];
     for switch in route.switches_directions.keys() {
-        if !route_path.switches_directions.contains_key(switch) {
+        if !route_path
+            .switches_directions
+            .iter()
+            .any(|(k, _)| k == switch)
+        {
```

Comment:
We should make a `HashSet` to decrease the complexity from `O(N²)` to `O(N)`

Reply:
Hopefully that'll solve your `utoipa` issue as well 🤞